proxyd/consensus_poller.go

```diff
@@ -318,18 +336,22 @@ func (cp *ConsensusPoller) UpdateBackend(ctx context.Context, be *Backend) {
 	}
 
 	latestBlockNumber, latestBlockHash, err := cp.fetchBlock(ctx, be, "latest")
-	if err != nil {
-		log.Warn("error updating backend - latest block", "name", be.Name, "err", err)
+	if err != nil || latestBlockNumber == 0 {
```

Comment:
when block is 0, we should increment `networkRequestsSlidingWindow`

Reply:
Happy to rename the var. 

Just to clarify on any non 200 the networkErrorsSlidingWindow is incremented within the fetchBlock function already

This update would mean the sliding window would then be incremented twice if a blockNumber 0 came in or a non-200. 

Please confirm if this is desired.

Reply:
but that would return an `err` right?

we can avoid increment dupes this way:

```
if err != nil {
 // handle error and return
} 

// the statement below will only be reached in case there is no error
if blockHeight == 0 {
  // increment the window and return
}  
```